modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/__artifactId__.es.js

```diff
@@ -1,6 +1,14 @@
+#parse ("definitions.vm")
```

Comment:
Hey @gabrielibson, you just need to add this line if you'll use some variable of this file, like `formFieldTypeName`, but apparently this is not the case, right? If so, please remove this line.